docs/userguide/envvars.md

```diff
@@ -74,6 +74,18 @@ are being shipped in the default dependencies because there is no good way of cr
 opt-out dependencies, but the baybe package will work without them.
 ```
 
+## Polars
+If baybe was installed with the additional Polars dependency (`baybe[polars]`), it will
+use the advanced methods of Polars to create the searchspace lazily and perform a
+streamed evaluation of constraints. This will improve speed and memory consumption
+during this process, and thus might be beneficial for very large search spaces.
+
+Since this is still somewhat experimental, you might want to deactivate polars without
+changing the Python environment. To do so, you can set the following environment
+variable to any value:
+- `BAYBE_DEACTIVATE_POLARS`
```

Comment:
Why have a one point list here? I'd prefer "you can set the environment variable`.......` to an arbitrary value"

Reply:
makes sense, changed